functions/jest.config.json

```diff
@@ -1,6 +1,4 @@
 {
-  "globalSetup": "<rootDir>/global-setup.ts",
-  "globalTeardown": "<rootDir>/global-teardown.ts",
```

Comment:
why are we also removing global jest setup and teardown?

Reply:
These have been replaced in ci by start-server-and-test. Manual testing now requires running start:cloud and test:cloud separately. This is a more reliable way of making sure the server is up, and allows for more iterative testing (via fast-refresh server).